userdocs/profiles.dev/package.json

```diff
@@ -36,5 +38,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "json-loader": "^0.5.7",
+    "raw-loader": "^4.0.2"
```

Comment:
don't think we require these dependencies 🤔

Reply:
i didn't know what they did, assumed you added them so left them alone 😅